Shooting/Shooting/Bullet/Bullet.cpp

```diff
@@ -0,0 +1,22 @@
+﻿#include"Bullet.h"
+
+
+Bullet::Bullet(POSITION pPlayePosition)
```

Comment:
仮引数、ポインタじゃないので
```suggestion
Bullet::Bullet(POSITION playePosition)
```
では？

Reply:
弾クラスのコンストラクタ変更で直しました